sdks/java/fn-execution/src/main/java/org/apache/beam/sdk/fn/data/BeamFnDataOutboundAggregator.java

```diff
@@ -157,15 +161,18 @@ public void flush() throws IOException {
     if (elements.getDataCount() > 0 || elements.getTimersCount() > 0) {
```

Comment:
Lets make flush private and/or package private marked with `@VisibleForTesting` if used in tests.